lib/faexport.rb

```diff
@@ -400,6 +400,219 @@ def ensure_login!
       end
     end
 
+    # GET /new_submissions.json
+    get %r{/new_submissions\.(json|xml|rss)} do |type|
+      ensure_login!
+      set_content_type(type)
+      from_id = params['from']
```

Comment:
You should check to make sure this matches `ID_REGEX` and fail if it doesn't.

Reply:
Sure, adding that check in 8656ab05d82aa3999f9b6d0b21815cab1eec99c7 to mirror the check in GET /user/{name}/gallery|scraps|favorites
Is that good?

Reply:
Yep